fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBIndexOutput.java

```diff
@@ -217,7 +196,7 @@ private void flush() {
             buffer.flip();
             byte[] arr = new byte[buffer.remaining()];
             buffer.get(arr);
-            flushes.add(fdbDirectory.writeData(id, ( (currentSize - 1) / blockSize), arr));
+            actualSize += fdbDirectory.writeData(id, ((currentSize - 1) / blockSize), arr);
```

Comment:
This makes the class non-thread-safe. Are we OK with that at this point?

Reply:
Yes. This writing module, by definition, cannot be used asynchronously - as the written data must be consistent.